volume/nfs/nfs.go

```diff
@@ -157,7 +157,14 @@ func (d *NFSDriver) Status() (*volume.Status, error) {
 
 // Release implements volume.Driver.Release
 func (d *NFSDriver) Release(volumeName string) error {
-	return ErrNotSupported
+	volumePath := filepath.Join(d.root, volumeName)
```

Comment:
This is a nit - but there's no need to create `volumePath` if the network is disabled. Can you move inside the `if` block please?